contributing/getting-started-e2e.md

```diff
@@ -151,16 +151,14 @@ Now run `git remote -v` again and notice that you have set up your upstream alia
 
 Now that you have all of the files copied locally you can actually build the code and run a local server to try things out.
 
-amphtml uses Node.js, the yarn package manager and the Gulp build system to build amphtml and start up a local server that lets you try out your changes.  Installing these and getting amphtml built is straightforward:
+amphtml uses Node.js, the npm package manager and the Gulp build system to build amphtml and start up a local server that lets you try out your changes.  Installing these and getting amphtml built is straightforward:
 
 * Install [NodeJS](https://nodejs.org/) (which includes npm).
 
-* Install [yarn](https://yarnpkg.com/en/docs/install)
+* Install [npm](https://www.npmjs.com/get-npm)
+
+* In your local repository directory (e.g. `~/src/ampproject/amphtml`), install the packages that AMP uses by running `npm install`
```

Comment:
Make this consistent with what's below re: `sudo`. Also, for commands that will likely be copy-pasted by new developers, let's put them on a separate line, like it previously was.

Reply:
Done